### DISCUSSION TITLE
Scrobbling support using last.fm for android!

### DISCUSSION THREAD

**Discussion by Vjay15** [KRTirtho/spotube#841] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

I really wish there was scrobbling support for android just like the desktop version, but i couldnt find that option in the settings of the android app :(.

### Suggested solution

i really wish i could give a suggested solution but im not that savvy enough yet ;-;

**Comment by KRTirtho** [comment]
Scrobbling support was added for all platforms together. Just sign in with your LastFm account and that's it
